e2e/testcases/managed_resources_test.go

```diff
@@ -806,6 +810,24 @@ func TestDriftKubectlAnnotateManagedFieldWithIgnoreMutationAnnotation(t *testing
 	// The reason we need to stop the webhook here is that the webhook denies a request to modify Config Sync metadata
 	// even if the resource has the `client.lifecycle.config.k8s.io/mutation` annotation.
 	nomostest.StopWebhook(nt)
+	// Stopping the webhook causes the reconciler to restart. Wait so that we aren't
+	// racing with the applier and are actually testing the remediator.
+	tg := taskgroup.New()
+	tg.Go(func() error {
+		predicates := []testpredicates.Predicate{
+			testpredicates.StatusEquals(nt.Scheme, kstatus.CurrentStatus),
+			testpredicates.DeploymentMissingEnvVar(reconcilermanager.Reconciler, reconcilermanager.WebhookEnabled),
+		}
+		return nt.Watcher.WatchObject(kinds.Deployment(),
+			core.RootReconcilerName(rootSyncID.Name), configsync.ControllerNamespace, predicates)
+	})
+	tg.Go(func() error {
+		return nt.Watcher.WatchObject(kinds.Namespace(), "bookstore", "",
+			[]testpredicates.Predicate{
+				testpredicates.HasAnnotation("season", "summer"),
```

Comment:
The assertion proves that only the remediator respects the ignore mutation annotation, while the applier does not.
Suggest adding a comment that the applier reverts the mutation.

Reply:
done